docs/source/links/rest.md

```diff
@@ -0,0 +1,148 @@
+---
+title: apollo-link-rest
+description: Call your REST APIs inside your graphQL queries.
+---
+
+[**Read the announcement post! ( to be pusblished soon )
+🎉**](https://dev-blog.apollodata.com/)
+
+If you have a REST API, you can now query your endpoints using GraphQL and have all your data managed by Apollo Client. There is no need to write a wrapper around your REST API.
+
+<h2 id="start">Quick start</h2>
+
+To get started, install `apollo-link-rest` from npm:
+
+```bash
+npm install apollo-link-rest --save
+```
+
+The rest of the instructions assume that you have already [set up Apollo
+Client](/docs/react/basics/setup.html#installation) in your application. After
+you install the package, you can create your rest link.
+
+```js
+const link = new RestLink({ uri: "https://api.example.com" });
```

Comment:
Needs more detail about the ordering of the links. Your rest link should probably come after your state link, but before your HttpLink if you're using it in conjunction with a GraphQL server.